Documentation/CRDs/ceph-nfs-crd.md

```diff
@@ -204,3 +204,43 @@ the size of the cluster.
     ceph orch set backend ""
     ceph mgr module disable rook
     ```
+
+## External Clusters
+
+Make use of the NFS client running on the external ceph standalone cluster.
+
+Create the export,
+```console
+ceph nfs export create cephfs <nfs-client-name> /test <filesystem-name>
+```
+
+Create the PV,
+```console
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: <pv-name>
+spec:
+  capacity:
+    storage: <storage>
+  accessModes:
+  - ReadWriteOnce
+  nfs:
+    path: /test
+    server: <nfs-client-server-ip>
+  persistentVolumeReclaimPolicy: Retain
+```
+And PVC,
+```
```

Comment:
Always need a blank line before the start of a quoted section or else it won't render correctly
```suggestion

```yaml
```